src/univers/version_range.py

```diff
@@ -899,6 +899,56 @@ def from_native(cls, string):
         return cls(constraints=constraints)
 
 
+vers_by_github_native_comparators = {
+    "==": "=",
+    "<=": "<=",
+    ">=": ">=",
+    "<": "<",
+    ">": ">",
+}
+
+
+def build_constraint_from_github_advisory_string(scheme: str, string: str):
+    """
+    Return a VersionConstraint built from a single github-native version
+    relationship ``string``.
+
+    >>> vr = build_constraint_from_github_advisory_string("gem","<= 2.24")
+    >>> assert str(vr) == "<=2.24", str(vr)
+    >>> vr = build_constraint_from_github_advisory_string("pypi","< 9.0")
+    >>> assert str(vr) == "<9.0", str(vr)
+    """
+    vrc = RANGE_CLASS_BY_SCHEMES.get(scheme)
```

Comment:
IMHO we can be sure that scheme exists:
```suggestion
    vrc = RANGE_CLASS_BY_SCHEMES[scheme]
```